src/app/components/server-compare-charts/server-compare-charts.component.ts

```
@@ -0,0 +1,1076 @@
+      }
+    });
+    return isBest ? this.bestCellStyle : '';
+  }
+
+  isBestStyle(value: any, values: any[], benchmark: any) {
+    if(value === '-' || value === 0) return '';
+    let isBest = true;
+
+
+    values.forEach((v) => {
+      if(!isNaN(v)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)